demo/api/src/config/config.ts

```diff
@@ -18,6 +18,7 @@ export function createConfig(processEnv: NodeJS.ProcessEnv) {
         apiUrl: envVars.API_URL,
         apiPort: envVars.API_PORT,
         corsAllowedOrigins: envVars.CORS_ALLOWED_ORIGINS.split(","),
+        hmacSecret: envVars.HMAC_SECRET,
```

Comment:
did you consider generating a secret on startup? (with persistance in db/pvc across startups)

Reply:
We currently don't have a mechanism to automatically create secrets for deployments. However, for our needs a rolling secret is sufficient here as the validation is always predeced by the creation of it. So I removed the necessity of having a persistent secret here.

Reply:
Correction: it has to be the same across multiple api-pods. I added it.